internal/test/wsu/wsu_test.go

```diff
@@ -244,11 +246,15 @@ func runE2ETestSuite(t *testing.T, vm e2ef.WindowsVM, ansibleOutput string) {
 	tempDirPath, err := getAnsibleTempDirPath(ansibleOutput)
 	require.NoError(t, err, "Could not get path of Ansible temp directory")
 
+	binaryFileList := []string{"kubelet.exe", "worker.ign", "wmcb.exe", "hybrid-overlay.exe", "kube.tar.gz"}
+	hybridOverlaylogList := []string{"hybridOverlayStdout.log", "hybridOverlayStderr.log"}
+	kubeProxylogList := []string{"kube-proxy.exe.INFO", "kube-proxy.exe.WARNING"}
```

Comment:
Isn't there any error file generated for kube-proxy?

Reply:
missed this, as it wasn't created for one of the tests I ran, can verify again and update.

Reply:
yes because that file is not always present in the directory.